docs/glossary.md

```diff
@@ -173,6 +173,17 @@ A tree object represents a snapshot of a directory in the repository. Tree
 objects are defined recursively; each tree object only has the files and
 directories contained directly in the directory it represents.
 
+## Tracked branches and tracking branches
+
+A remote branch can be made "tracked" with the `jj branch track` command. This
+results in a "tracking" local branch that tracks the remote branch.
+
+Both of these are different from Git's notion of "remote-tracking branches",
+though `jj` has an analogue to that as well.
```

Comment:
Nit: Either I'm stupid and don't get the analogue from the existing text or this sentence lacks context. If you want to keep the sentence, refering to the analogue would be helpful.

Reply:
Good catch; I got rid of it.